src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleList } from '../PeopleList';
+import { useParams } from 'react-router-dom';
+import React from 'react';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  const { personId = '' } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
```

Comment:
In the catch block, you should also catch a possible error, because now the error does not look informative, for example, you can do this:

```js

catch(error) { setIsError(`Something went wrong: ${error.message}`)}
```